### PR TITLE
list component

### PR DESCRIPTION
**Pull request by prakashsakari**
This PR has the following

1. Ordered List
2. Unordered List
3. Nested List
4. Stacked List
5. Stacked List with Icon
l1
l2
l3

### CODE REVIEW

File: List/list.css [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    width: 20px;
+    margin-left: auto;
+}
+
+.margin-16px{
+    margin: 1rem 0;
+}
+
+.hero-drawer {
+    width: 15.3125rem;
```
Comment (khushijohri2001) [inline List/list.css:70]: So precise with the values😂 Nice!!

**Review by khushijohri2001 — APPROVED** [review]
Like the naming consistency 👍

File: List/list.css (hunk 1 above)
Comment (prakashsakari) [reply List/list.css:70]: hahaha, thanks

File: List/list.html [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+                        alt="image">
+                </div>
+                <div class="name d-flex align-center justify-center">
+                    <span>List Item 1</span>
+                    <input class="check-input" type="checkbox">
+                </div>
+            </div>
+            <div class="d-flex align-center gap margin-16px">
+                <div class="img-container">
+                    <img class="avatar-image"
```
Comment (JaswanthReddyPadala) [inline List/list.html:117]: Have you used same avatar component from your CL?

Comment (prakashsakari) [reply List/list.html:117]: no, that has some other dimensions

**Review by JaswanthReddyPadala — APPROVED** [review]
Code and UI looks Clean,
Good to Merge ✅

**Pull request merged**